rusoto/credential/src/lib.rs

```diff
@@ -507,16 +507,14 @@ mod tests {
     use super::*;
 
     #[test]
-    fn credential_chain_explicit_profile_provider() {
+    fn profile_provider() {
```

Comment:
I think we're losing a bit of context here with the shortening of this name.

What do you think about:

`profile_provider_credentials()`